src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,24 +1,34 @@
 package edu.kis.vh.nursery;
 
+import edu.kis.vh.nursery.storage.IntLinkedList;
+import edu.kis.vh.nursery.storage.IntStorageInterface;
+
 public class FIFORhymer extends DefaultCountingOutRhymer {
 
-    private final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+    private final IntLinkedList TEMP = new IntLinkedList();

```

Comment:
This object is not a constant (it is mutable), the name should be written lowercase.